app/models/day.rb

```diff
@@ -0,0 +1,14 @@
+class Day
```

Comment:
There's no spec for this model. I know it may seem silly, but I feel we should have models covered.